modules/apps/object/object-js-components-web/src/main/resources/META-INF/resources/components/Select/SingleSelect.tsx

```diff
@@ -49,6 +55,18 @@ export function SingleSelect<
 					{option.description && (
 						<span className="text-small">{option.description}</span>
 					)}
+
+					{Liferay.FeatureFlags['LPS-162133'] && option.type && (
+						<ClayLabel
+							displayType={
+								option.type === 'email' ? 'success' : 'info'
+							}
+						>
+							{option.type === 'email'
+								? Liferay.Language.get('email')
+								: Liferay.Language.get('user-notification')}
+						</ClayLabel>
+					)}
```

Comment:
Hey @alinedoleron, since the `SingleSelect` component is a generic component I think it's a good idea to pass this `ClayLabel` which is specific to Notification, as a `children`, so we keep the component as generic as possible.